content/blogposts/2022/sourcegraph-non-devs.md

```diff
@@ -90,10 +90,10 @@ Then, I:
 
 A customer was seeing incomplete result counts for the search results returned for a Code Insights feature, making it impossible to reason about their code based on these incomplete results.
 
-**Search:** A term like “limit 5000” is likely to bring up many hits throughout our codebase, so I used [a diff search with a timeframe](https://sourcegraph.com/search?q=context:global+repo:%5Egithub%5C.com/sourcegraph/sourcegraph%24+type:diff+limit+5000+after:%22five+months+ago%22&patternType=regexp) to scope my search to the time the behavior started appearing.
+**Search:** A term like “limit 5000” is likely to bring up many hits throughout our codebase, so I used [a diff search with a timeframe](https://sourcegraph.com/search?q=repo:%5Egithub%5C.com/sourcegraph/sourcegraph%24%403.36+type:diff+limit+5000+after:%222021-06-01%22+before:%222021-09-01%22&patternType=regexp) to scope my search to the time the behavior started appearing.
 
 <p style={{ width: '100%' }}>
-    <iframe src="https://sourcegraph.com/embed/notebooks/Tm90ZWJvb2s6MTc5?theme=light" frameBorder="0" sandbox="allow-scripts allow-same-origin allow-popups" style={{ width: '100%', height: '300px' }}></iframe>
+    <iframe src="https://sourcegraph.com/embed/notebooks/Tm90ZWJvb2s6MTc1NQ==?theme=light" frameBorder="0" sandbox="allow-scripts allow-same-origin allow-popups" style={{ width: '100%', height: '300px' }}></iframe>
```

Comment:
This iframe is showing up not found. Are you seeing that too? @Joelkw 
<img width="820" alt="image" src="https://user-images.githubusercontent.com/59381432/195757769-9175868a-1ab2-459e-8468-d76b41de2c2f.png">

Reply:
Ah @st0nebraker good catch! I think it's a permissions thing because it shows for me – just updated it to be public, try now?

Reply:
Great! I sees it now 👀 👍🏻